.gitmodules

```
@@ -0,0 +1,3 @@
+[submodule "providers"]
+	path = providers
+	url = git@github.com:Materials-Consortia/providers.git
```

Are the tabs needed?
```suggestion
path = providers
url = git@github.com:Materials-Consortia/providers.git
```

---

This was set by git when I added the submodule. So I don't know, but I don't want to affect what git itself does.